common/server-sent-events.conf

```diff
@@ -1,8 +1,3 @@
-proxy_buffering off;
```

Comment:
@malnick Can we please rename this file to i.e. `http11_upstream.conf`? With buffering it is not SSE anymore.